clang/lib/CIR/CodeGen/CIRGenBuiltin.cpp

```diff
@@ -382,3 +382,13 @@ void CIRGenFunction::emitVAStartEnd(mlir::Value argValue, bool isStart) {
   else
     cir::VAEndOp::create(builder, argValue.getLoc(), argValue);
 }
+
+// FIXME(cir): This completely abstracts away the ABI with a generic CIR Op. We
```

Comment:
I think this comment doesn't belong. Completely abstracting away the ABI details is what we want in CIR. At most, this should be a comment saying that the ABI details will be handled in LoweringPrepare.